buildstockbatch/local.py

```diff
@@ -393,7 +393,13 @@ def main():
     )
     group.add_argument(
         "--uploadonly",
-        help="Only upload to S3, useful when postprocessing is already done. Ignores the " "upload flag in yaml",
+        help="Only upload to S3, useful when postprocessing is already done. Ignores the upload flag in yaml."
+        " Errors out if files already exists in s3",
+        action="store_true",
+    )
+    group.add_argument(
+        "--continue_upload",
+        help="Continue uploading to S3, useful when previous upload was interrupted.",
```

Comment:
Why have two flags instead of wrapping this into the `--upload_only_` flag?

Reply:
I added a new flag to preserve the existing behavior of --upload_only, which is to error out if there are existing files in s3.